README.md

```diff
@@ -1,10 +1,14 @@
 # NHS Wales API Catalogue
 
+The files in this repository are used to generate documentation and user guides for NHS Wales APIs.
```

Comment:
How about using an active voice throughout..._We use the files in this repository to generate documentation and user guides for NHS Wales APIs_.